ivy/functional/frontends/paddle/math.py

```diff
@@ -193,6 +193,11 @@ def divide(x, y, name=None):
 def erf(x, name=None):
     return ivy.erf(x)
 
+@with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
+@to_ivy_arrays_and_back
+def erfinv(x, name=None):
+    return  ivy.erfinv(x)
```

Comment:
There's no `ivy.erfinv` function. You will need to implement `erfinv` using a composition of other `ivy` functions.